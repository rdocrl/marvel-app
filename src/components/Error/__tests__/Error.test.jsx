import { render } from '@testing-library/react';
import ErrorComponent from '../';

describe('<ErrorComponent /> tests', () => {
  describe('Snapshots', () => {
    it('Should render', () => {
      const { asFragment } = render(<ErrorComponent />);
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
