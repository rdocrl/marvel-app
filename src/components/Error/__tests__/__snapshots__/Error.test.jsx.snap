// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorComponent /> tests Snapshots Should render 1`] = `
<DocumentFragment>
  <h2>
    Something went wrong
  </h2>
  <img
    alt="Something went wrong"
    src="not-found.png"
    width="300"
  />
</DocumentFragment>
`;
